integrationTests/DonationRepositoryTest.php

```diff
@@ -214,7 +216,9 @@ public function testPushSalesForcePendingPushesNewDonationButNotVeryNewDonation(
 
         $sut = $this->getService(DonationRepository::class);
         $donationClientProphecy = $this->prophesize(\MatchBot\Client\Donation::class);
-        /** @var Donation $donationInDB */
+
+        $busProphecy = $this->prophesize(RoutableMessageBus::class);
+        $dummyEnvelope = new Envelope(new \stdClass()); // Final so can't prophesise.
```

Comment:
yes, `\Symfony\Component\Messenger\Envelope` is final but as its a value object I think there's no reason we'd want to prophesise it.

Also $dummyEnvelope is only used once, probably simpler to inline it?

Reply:
Done in c0d6d320d245fad4e77560e4cc4ff68e6d4fc19a